QuickBooksSharp/Webhooks/EventNotification.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace QuickBooksSharp
+{
+    public class EventNotification
+    {
+        public string RealmId { get; set; } = string.Empty;
```

Comment:
Assignments are not necessary, are they? If it's to handle nullability check, I think you should use  = default!

Reply:
Same thing.

- **QuickBooksSharp.csproj** does have `<Nullable>enable</Nullable>`. 

Not setting the default value generates this warning:
```
Warning	CS8618	Non-nullable property 'Id' must contain a non-null value when exiting constructor. Consider declaring the property as nullable.	QuickBooksSharp	C:\Repos\QuickBooksSharp\QuickBooksSharp\Webhooks\EntityChange.cs	18	Active
```

Reply:
Same as above, please assign `default!`